custom/icds/location_reassignment/parser.py

```diff
@@ -419,87 +422,90 @@ def valid_transitions_json(self, for_location_type=None):
         return json_response
 
 
-class HouseholdReassignmentParser(object):
+class CaseReassignmentParser(metaclass=ABCMeta):
+    case_id_column = None
+    new_site_code_column = None
+
     def __init__(self, domain, workbook):
         self.domain = domain
         self.workbook = workbook
-        self.reassignments = {}  # household id mapped to a dict with old_site_code and new_site_code
+        self.reassignments = {}  # case id mapped to a dict with old_site_code and new_site_code
```

Comment:
`case` was used instead of `household` to bring commonality, which should be okay.